includes/class-wcs-template-loader.php

```diff
@@ -289,4 +357,53 @@ public static function get_opc_subscription_add_to_cart() {
 		global $product;
 		wc_get_template( 'checkout/add-to-cart/simple.php', array( 'product' => $product ), '', PP_One_Page_Checkout::$template_path );
 	}
+
+	/**
+	 * Handles relocated templates.
+	 *
+	 * Hooked onto 'wc_get_templates'.
+	 *
+	 * @since 1.5.0
+	 */
+	public static function handle_relocated_templates( $template, $template_name, $args, $template_path, $default_path ) {
+		if ( ! $default_path || ! in_array( $template_name, self::$relocated_templates, true ) ) {
+			return $template;
+		}
+
+		$subscriptions_core_templates_path = WC_Subscriptions_Core_Plugin::instance()->get_subscriptions_core_directory( 'templates/' );
+		if ( $default_path === $subscriptions_core_templates_path ) {
+			return $template;
+		}
+
+		$is_default_path_wcs = false;
+		if ( class_exists( 'WC_Subscriptions_Plugin' ) ) {
+			// WCS 4+ is active.
+			$wcs_templates_path  = WC_Subscriptions_Plugin::instance()->get_plugin_directory( 'templates/' );
+			$is_default_path_wcs = $default_path === $wcs_templates_path;
+		} elseif ( class_exists( 'WC_Subscriptions' ) ) {
+			// WCS 3 or older is active.
```

Comment:
I'm not sure if this block is necessary because there will be no case where subscriptions-core is loaded when WCS 3 or older is active, right?

Reply:
Yeah that's correct! We don't need this `elseif` for the `class_exists( 'WC_Subscriptions' )` case.